GameServerCore/Domain/GameObjects/Spell/Missile/ISpellLineMissile.cs

```diff
@@ -0,0 +1,10 @@
+﻿using GameServerCore.Enums;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace GameServerCore.Domain.GameObjects.Spell.Missile
+{
+    public interface ISpellLineMissile : ISpellCircleMissile
```

Comment:
Why line is a type of circle?

Reply:
Line is referred to as Arc in content. Arc missiles can hit multiple targets, can follow a set path, can be attached to a unit, or both. I just inherited CircleMissile because it inherits ISpellMissile and it has a Destination and ObjectsHit list.